templates/README.template

```diff
@@ -1,112 +1,216 @@
+      }
+    ]
+  }
+}
+```
+
+<!-- isPlugin ? use(this) : delete(isLoader) -->
+**webpack.config.js**
+```js
+module.exports = {
+  plugins: [
+    new ${name}Plugin(options)
+  ]
+}
+```
+
+<h2 align="center">Options</h2>
+
+|Name|Type|Default|Description|
+|:--:|:--:|:-----:|:----------|
+|**`option1`**|`{String}`|`/`|Lorem ipsum dolor sit amet...|
+|**`option2`**|`{Boolean}`|`true`|Lorem ipsum dolor sit amet...
+|**`option3`**|`{Function\|Object}`|`{}`|Lorem ipsum dolor sit amet...|
 
-<h2 align="center">Examples</h2>
 
+### `Option 1`
+
+Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+
+<!-- isLoader ? use(this) : delete(isPlugin) -->
+**webpack.config.js**
+```js
+{
+  test: /\.ext$/,
+  use: [
+    {
+      loader: `${name}-loader`,
+      options: { option1: '' }
+    }
+  ]
+}
+```
+
+<!-- isPlugin ? use(this) : delete(isLoader) -->
+**webpack.config.js**
+```js
+new ${name}Plugin({ option1: '' })
+```
+
+### `Option 2`
+
+Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+
+<!-- isLoader ? use(this) : delete(isPlugin) -->
 **webpack.config.js**
+```js
+{
+  test: /\.ext$/,
+  use: [
+    {
+      loader: `${name}-loader`,
+      options: { option2: '' }
+    }
+  ]
+}
+```
 
+<!-- isPlugin ? use(this) : delete(isLoader) -->
+**webpack.config.js**
+```js
+new ${name}Plugin({ option2: '' })
+```
+
+### `Option 3`
+
+Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+
+<!-- isLoader ? use(this) : delete(isPlugin) -->
+**webpack.config.js**
+```js
+{
+  test: /\.ext$/,
+  use: [
+    {
+      loader: `${name}-loader`,
+      options: { option3: '' }
+    }
+  ]
+}
+```
+
+<!-- isPlugin ? use(this) : delete(isLoader) -->
+**webpack.config.js**
+```js
+new ${name}Plugin({ option3: '' })
+```
+
+<h2 align="center">Examples</h2>
+
+### `Example 1`
+
+Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+
+**webpack.config.js**
 ```js
 // Loader/plugin setup here..
 ```
 
 **file.ext**
-
 ```js
 // Source code here...
 ```
 
 **bundle.js**
-
 ```js
-require("${package}!./file.EXT");
-
 // Bundle code here...
 ```
 
 <h2 align="center">Maintainers</h2>
 
-```bash
-https://api.github.com/users/MAINTAINER
-```
-
+<!--replace 'webpack-bot' with Github User -->
+<!--replace Name with the first & last name of the Github User -->
 <table>
   <tbody>
     <tr>
       <td align="center">
-        <a href="https://github.com/">
-          <img width="150" height="150" src="https://avatars.githubusercontent.com/u/5419992?v=3&s=150">
-          </br>
-          Name
-        </a>
+        <img width="150" height="150"
```

Comment:
@d3viant0ne This was my first intention aswell, but I wasn't sure if everyone would agree with that, so I used to non-personal bot account, will update